### PR TITLE
Get request

### PR DESCRIPTION
**Pull request by KStockley**
My server GET route is up for review when someone gets the chance.
@pickles27 @jordanstholmes @edenlhn

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -21,7 +21,6 @@ coverage
 .grunt
 
 # Compiled Dirs (http://nodejs.org/api/addons.html)
-build/
-dist/
+public/bundle.js
```
Comment (jordanstholmes) [inline .gitignore]: Nice on the ignoring the bundle :)

File: server/server.js [hunk 2]
```diff
@@ -1,14 +1,25 @@
...
 const app = express();
 
 app.use(express.static(path.resolve(__dirname, '../../funding-widget/public')));
 app.use(express.json());
 app.use(express.urlencoded({ extended: true }));
 
+app.get('/campaigns/:campaignId/stats', (req, res) => { // handle requests for a given campaign's stats
+  let campaign = parseInt(req.params.campaignId); // parse out the campaign's unique id
+
+  db.query(statsQuery, [campaign], (error, results) => { // retrieve campaign stats from the database
```
Comment (jordanstholmes) [inline server/server.js]: I second Sarah's comment about putting some error handling in here (I wouldn't have noticed it on my own).

File: server/server.js [hunk 3]
```diff
@@ -1,14 +1,25 @@
...
+    res.status(200).send(results[1][0]);
+  });
+});
+
 app.listen(port, () => {
   console.log(`Listening on port: ${port}`);
```
Comment (jordanstholmes) [inline server/server.js]: nice factoring out the port :) which reminds me, I believe we all should be using different ports. Let's talk about it at next standup?

File: server/server.js (hunk 2 above)
Comment (KStockley) [reply server/server.js]: I have added an if() block to log any errors that come through on the database query. Thanks @pickles27 and @jordanstholmes for catching that.

File: server/server.js (hunk 3 above)
Comment (KStockley) [reply server/server.js]: Good idea.

**Pull request merged**